python/trifinger_simulation/pinocchio_utils.py

```diff
@@ -3,14 +3,11 @@
 import pinocchio
 
 
-class PinocchioUtils:
-    """
-    Consists of kinematic methods for the finger platform.
-    """
+class Kinematics:
+    """Forward and inverse kinematics for the finger platform."""
```

Comment:
Just a minor comment, but I think maybe it would be better to call it the TriFinger platform instead of the finger platform. We could start doing this now with new changes

Reply:
I wouldn't call it "TriFinger" here, as this class should work with any number of fingers.  I can make the latter explicit in the doc, though.

Reply:
I just noticed that the docstrings are of this class are outdated in a few places.  Will update everything.